monero-outreach-docs/translations/pt-br/archive/rpc_pay_pt-br.md

```diff
@@ -0,0 +1,133 @@
+
+O Primo oferece compensação para o dono do site e para os usuários que evitam anúncios, tudo isso enquanto fortalece a rede Monero. Os clientes fornencendo hashes e os servidores usando essas hashes formam um processo de mineração completo. E quanto maior a diversidade de mineradores de Monero, melhor. O Primo é um exemplo excelente de possibilidades para RPC-Pay.
+
+## _Olhando para frente_
+
+O RPC-Pay foi lançado a primeira vez com a versão Monero 0.15. É novinho. Com o tempo, recursos serão adicionados, e conforme o RPC-Pay for evoluindo, suas aplicações também evoluem. O Primo é a primeira aplicação de fora, servindo como um exemplo para os outros. Qualquer servidor na Internet fornecendo dados ou cálculos para clientes é um candidato potencial para usar o RPC-Pay para arrecadar dinheiro. Com o RandomX como algoritmo de prova de trabalho do Monero, também novo na versão 0.15, qualquer computador pode calcular hashes de maneira eficiente para fazer pagamentos RPC-Pay. O momento é certo para esse novo recurso e ele provocará muitos desenvolvimentos interessantes.
+
+### _Guias de introdução_
+
+#### _#1: Exemplo de RPC Monero_
+---
+
+Usando a linha de comando curl ([curl.haxx.se](https://curl.haxx.se/)), você pode enviar chamadas de RPC Monero tanto para sua própria instância _monerod_ como para um servidor público. Por exemplo, se você quiser saber quantos blocos existem na blockchain do Monero, você poder usar o comando RPC _get_block_count_. Para ter essa informação de um servidor público no Mundo Monero, instale o curl e na linha de comando, escreva o seguinte:
+
+```
+curl -X POST [uwillrunanodesoon.moneroworld.com:18089/json_rpc](http://uwillrunanodesoon.moneroworld.com:18089/json_rpc) -H 'Content-Type:application/json' --data "{\"jsonrpc\":\"2.0\",\"id\":\"0\",\"method\":\"get_block_count\"}"
+```
+
+Você terá uma resposta mais ou menos assim:
+
+```
+{ "id": "0", "jsonrpc": "2.0", "result": { "count": 2021560, "status": "OK", "untrusted": false } }
+```
+
+O número de blocos na blockchain quando isso foi rodado era de 2.021.560.
+
+Se você estiver rodando sua própria instância _monerod_ com uma configuração padrão, você conseguirá a mesma informação com o seguinte comando, muito similar, no mesmo computador:
+
+```
+curl -X POST [127.0.0.1:18081/json_rpc](http://127.0.0.1:18081/json_rpc) -H 'Content-Type:application/json' --data "{\"jsonrpc\":\"2.0\",\"id\":\"0\",\"method\":\"get_block_count\"}"
+```
+
+Para mais informações sobre a interface _monerod_ RPC, veja o [www.getmonero.org/resources/developer-guides/daemon-rpc.html](https://www.getmonero.org/resources/developer-guides/daemon-rpc.html). E para informações sobre a interface carteira RPC, veja [www.getmonero.org/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/resources/developer-guides/wallet-rpc.html).
```

Comment:
```suggestion
Para mais informações sobre a interface _monerod_ RPC, veja o [www.getmonero.org/resources/developer-guides/daemon-rpc.html](https://www.getmonero.org/pt-br/resources/developer-guides/daemon-rpc.html). E para informações sobre a interface carteira RPC, veja [www.getmonero.org/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/pt-br/resources/developer-guides/wallet-rpc.html).
```

Both guides are not translated yet in getmonero.org but might be a good idea to link to them anyway.